### DISCUSSION TITLE
Does it accept tcp data sending ?

### DISCUSSION THREAD

**Discussion by Taymindis** [uNetworking/uWebSockets#899] [Q&A] [answered]
I saw there is a regular tcp-only app supported, does it accept tcp data sending from client side?

**Comment by ctubio** [comment]
nope (maybe ina future release), see https://github.com/uNetworking/uWebSockets/issues/791 or https://github.com/uNetworking/uWebSockets/search?q=client&type=Issues

**Comment by ghost** [comment, marked as answer]
You can do stuff with uSockets, Loop::get() returns an us_loop ptr
